src/parser/jobs/mnk/modules/PerfectBalance.tsx

```diff
@@ -0,0 +1,45 @@
+import {t, Trans} from '@lingui/macro'
+import React from 'react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import {Action} from 'data/ACTIONS/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import {Status} from 'data/STATUSES/STATUSES'
+import {BuffWindowModule} from 'parser/core/modules/BuffWindow'
+import {SEVERITY} from 'parser/core/modules/Suggestions'
+
+export default class PerfectBalance extends BuffWindowModule {
+	static handle = 'pb'
+	static title = t('mnk.pb.title')`Perfect Balance Windows`
+
+	buffAction: Action = ACTIONS.PERFECT_BALANCE
+	buffStatus: Status = STATUSES.PERFECT_BALANCE
+
+	expectedGCDs = {
+		expectedPerWindow: 6,
+		suggestionContent: <Trans id="mnk.pb.suggestions.missedgcd.content">
+			Try to land 6 GCDs during every <ActionLink {...ACTIONS.PERFECT_BALANCE} /> window.  If you cannot do this with full uptime and no clipping, consider adjusting your gearset for more Skill Speed.
+		</Trans>,
+		severityTiers: {
+			1: SEVERITY.MEDIUM,
+			3: SEVERITY.MAJOR,
+		},
+	}
+
+	trackedBadActions = {
+		icon: ACTIONS.TWIN_SNAKES.icon,
+		actions: [
```

Comment:
Should probably add Form Shift and Six-Sided Star to this. Possibly also Four-Point Fury. We have separate suggestions atm as part of the Greased Lightning module but I'm planning to gut that and only use it for Speedmod. Not sure how the icon would best be handled on this though.

Reply:
1. Six-Sided Star should be caught by missing the expected number of GCDs, unless it's the last one which can probably just be hand-waved as "we assume you know what you're doing and if you don't an actual human is gonna nitpick you."
2. Agree with Form Shift.
3. I'll follow up with Tiff and Aya re:recovery scenarios.

Reply:
Tiff agrees that Twin Snakes shouldn't be penalized.